### PR TITLE
Add nicer email templates and remove toggle switches.

### LINKED ISSUE #123: Make nice HTML template for emails
Emails work, but they're a little ugly at the moment. I'd like to make emails pretty.

### PR DESCRIPTION
**Pull request by tiffz**
I'll closed #123 after this although I know these aren't super fancy email templates yet. 

Screenshot of the checkboxes: https://www.dropbox.com/s/2gddj994c2j8rq4/Screenshot%202017-05-18%2023.02.16.png?dl=0

Toggle switches removed after a discussion with Noddy.

### CODE REVIEW

**Comment by NoiSek** [comment]
Thank the gods.

File: saylua/utils/email.py [hunk 1]
```diff
@@ -14,7 +16,8 @@ def send_email(to, subject, text):
         data={"from": app.config.get('AUTOSEND_EMAIL'),
```
Comment (NoiSek) [inline saylua/utils/email.py]: What the hell kind of format is this

Comment (tiffz) [inline saylua/utils/email.py]: Yeah I wanna rename it haha. Maybe EMAIL_SENDER?

Comment (NoiSek) [inline saylua/utils/email.py]: Well, that too, but I was referring your LISP-ish dict format with the inlined brackets.

**Pull request merged**